documentation/bolt_2_changes.md

```diff
@@ -0,0 +1,31 @@
+# Changes in Bolt 2.0
+
+## Output changes
+
+* JSON output now has a `target_count` field instead of `node_count`
+* Result objects in JSON output now contain keys called `target` and `value`, instead of `node` and `result`
+* The `prompt` plugin now prompts on stderr instead of stdout
+
+## CLI changes
+
+* The `--nodes` flag has been removed in favor of `--targets`
```

Comment:
It might be worth noting that `$nodes` and `$targets` parameters in plans will both still work with `--targets`